CHANGELOG.md

```diff
@@ -1,10 +1,10 @@
-# Unreleased
+# 78.1.0
 
 * Fix `bulk_unsubscribe` requires a `govuk_request_id` if you want to send a message
```

Comment:
Perhaps this should be addition?
KInda wierd, because without this additon its broken... so it is a fix 😖